src/vm/hints/hint_utils.rs

```diff
@@ -143,6 +144,42 @@ pub fn get_address_from_var_name(
     .ok_or(VirtualMachineError::FailedToGetIds)
 }
 
+//Gets the value of a variable name.
+//If the value is an MaybeRelocatable::Int(Bigint) return &Bigint
+//else raises Err
+pub fn get_integer_from_var_name<'a>(
+    var_name: &str,
+    ids: HashMap<String, BigInt>,
+    vm: &'a VirtualMachine,
+    hint_ap_tracking: Option<&ApTracking>,
+) -> Result<&'a BigInt, VirtualMachineError> {
+    let var_address = get_address_from_var_name(var_name, ids, vm, hint_ap_tracking)?;
+    vm.memory.get_integer(&var_address)
+}
+
+// Given a memory address and an offset
+// Gets the value of the address + offset
+//If the value is an MaybeRelocatable::Int(Bigint) return &Bigint
+//else raises Err
```

Comment:
Any chance to just make it receive a `Relocatable` instead?

Reply:
I made it this way so we avoid this boilerplate code in the hint function:

```
    let relocatable = if let MaybeRelocatable::RelocatableValue(relocatable) = addr{
        relocatable
    } else {
        return Err(VirtualMachineError::MemoryError(
            MemoryError::AddressNotRelocatable,
        ));
    };
```
Do you think it is better to receive a Relocatable as an input?

Reply:
In general, yes. But if you want we can leave it for a later refactor.